Drop-In/build.gradle

```diff
@@ -58,9 +58,18 @@ android {
 }
 
 dependencies {
-    api 'com.braintreepayments.api:braintree:3.15.0'
+    // TODO: consider making this api in core to allow for PaymentType and other @IntDef / @StringDef constants to work for merchants
+    implementation 'androidx.annotation:annotation:1.1.0'
+
+    api 'com.braintreepayments.api:braintree-core:4.0.0-beta4-SNAPSHOT'
+    api 'com.braintreepayments.api:three-d-secure:4.0.0-beta4-SNAPSHOT'
+    api 'com.braintreepayments.api:paypal:4.0.0-beta4-SNAPSHOT'
+    api 'com.braintreepayments.api:venmo:4.0.0-beta4-SNAPSHOT'
+    api 'com.braintreepayments.api:google-pay:4.0.0-beta4-SNAPSHOT'
+    api 'com.braintreepayments.api:card:4.0.0-beta4-SNAPSHOT'
+    api 'com.braintreepayments.api:data-collector:4.0.0-beta4-SNAPSHOT'
+    api 'com.braintreepayments.api:union-pay:4.0.0-beta4-SNAPSHOT'
```

Comment:
We should update the CHANGELOG to include that these are all now included by default in DropIn.

Reply:
Added this to the CHANGELOG